processing file asmFiles/test.ldst1.asm (1 of 1)
processing file asmFiles/test.ldst2.asm (1 of 1)
--- memsim.hex	2026-09-19 23:37:09.259411745 -0400
+++ memcpu.hex	2026-09-19 23:37:09.254411804 -0400
@@ -1,7 +1,6 @@
 :04000000FEEDB3B7A7
 :04000100FFFFF4B752
 :04000200F000621395
-:0400030040920233F2
 :040004008000651300
 :0400050040950533EA
 :0400060000EEF437DD
@@ -18,10 +17,6 @@
 :0400110000D52423CF
 :04001200007526232C
 :04001300FFFFFFFFED
-:040200000000733750
-:0402010000002701D1
-:0402020000001337AE
-:04020300FEEDBEEF5F
 :0403C000000073378F
 :0403C1000000270110
 :0403C20000001337ED
